pipeline/run-system-tests.sh

```diff
@@ -0,0 +1,3 @@
+sed -e "s|AWS_REGION|$AWS_REGION|" test/system/lib/config.js.sample > test/system/lib/config.js
+sed -e "s|DISCOVERY_SERVICE_ENDPOINT|$DISCOVERY_SERVICE_ENDPOINT|" test/system/lib/config.js.sample > test/system/lib/config.js
```

Comment:
It looks like the tests aren't running correctly in the pipeline yet; It could be because the config file here is typescript (.ts), rather than javascript (.js)